src/qililab/slurm.py

```diff
@@ -0,0 +1,62 @@
+import os
+from types import ModuleType
+
+from IPython.core.magic import needs_local_scope, register_cell_magic
+from IPython.core.magic_arguments import argument, magic_arguments, parse_argstring
+from submitit import AutoExecutor
+
+
+@magic_arguments()
+@argument("-o", "--output", help=("Name of the variables that will be returned from the SLURM job."))
```

Comment:
Suggestion of improvement of the docstring:
```suggestion
@argument("-o", "--output", help=("Output of the SLURM job. This name should correspond to a variable defined in the cell that we want to retrieve after execution. After queuing a cell, this variable will be converted to a `Job` class. To retrieve the results of the job, you need to call `variable.result()`."))
```